### ISSUE TITLE
appraisal.lic missing match strings related to getting/stowing weapons from a weapon harness

### ISSUE THREAD

**Issue by Nyctus** [rpherbig/dr-scripts#4833]
**Summary:**
When appraisal.lic pulls a weapon [diamondwood staff] from a weapon harness [embossed weapon harness] the messaging begins with "You deftly remove." This string does not appear to have a match that the script is listening for and so the script times out waiting for a matching string.

**Note:**
I encountered this issue on my Moon Mage.

**Repro:**
1. obtain an embossed weapon harness
2. wear the harness
3. obtain a diamondwood staff
4. put the diamondwood staff in the harness
5. ensure diamondwood staff and harness are listed in the relevant equipment sections of setup YAML
6. run ;appraisal in-game

**Results:**
When the diamondwood staff is drawn from the harness and held, the script will time out waiting for a match.

**Expected:**
The script is able to detect when the diamondwood staff is successfully removed from the weapon harness.

**Example game messaging:**

```
[appraisal]>get diamondwood.staff from my harness
You deftly remove the diamondwood staff from your harness.
>
[appraisal: *** No match was found after 15 seconds, dumping info]
[appraisal: messages seen length: 3]
[appraisal: message: Aura Sight  (12 roisaen)]
[appraisal: message: Braun's Conjecture  (56 roisaen)]
[appraisal: message: You deftly remove the diamondwood staff from your harness.]
[appraisal: checked against [/You get/i, /You pick/i, /You pluck/i, /You are already holding/i, /You fade in for a moment as you get/i, /I could not/i, /What were you/i, /Get what/i, /You need a free hand/i]]
[appraisal: for command get diamondwood.staff from my harness]
[appraisal]>get my diamondwood.staff
You are already holding that.
```

**Comment by BinuDR** [comment]
Are you a warrior mage by chance?

**Comment by Nyctus** [comment]
Moon Mage, sorry for leaving that out. I'll go mention it in the bug. Also, where can I get you the version info if needed?

**Comment by BinuDR** [comment]
It's fine, I'll fix the script.  I know some spells alter the wield/sheath messaging.  Can you also show me message for sheathing the staff?

**Comment by Nyctus** [comment]
Sure thing!
```>put staff in harness
You secure your diamondwood staff to your harness.```

**Comment by BinuDR** [comment]
Thanks

**Comment by Nyctus** [comment]
I am beginning to wonder if this is just a verby harness, here's the remove/wear text for the harness itself, which also caused some issues (I can write a separate bug for that if you like)

```
>remove harness
You unbuckle the straps of the harness and sling it off of your shoulder.
>wear harness
You toss one strap of the harness over your shoulder, pull it down and buckle it to its mate.
```

And here's the full name of the harness and epedia link:
**TAP** embossed weapon harness
**Full description:** A murder of crows are spread along the narrow harness in various stages of flight, their sleek bodies stamped in iridescent blue and black tones upon the dark ebony leather.  A pair of metal frogs are positioned along the weapon strap, allowing one to easily draw and store their weapons.
Link: https://elanthipedia.play.net/Item:Embossed_weapon_harness

**Comment by BinuDR** [comment]
Have you had any script try to remove or wear the harness and hang?  If so, let me know which scripts.

**Comment by Nyctus** [comment]
appraisal.lic hangs when removing and I think wearing it. 

I'll keep an eye out for others. Let me re-generate the errors and post them here.

**Removal**:
```
[appraisal]>remove my weapon.harness
You unbuckle the straps of the harness and sling it off of your shoulder.
```
```
[appraisal: message: You unbuckle the straps of the harness and sling it off of your shoulder.]
[appraisal: checked against [/^You .*(?i-mx:\bweapon.*\bharness)/, /^You (get|sling|pull|work|loosen|slide|remove|yank).*harness/, /Remove what/i, /You aren't wearing that/i, /slide themselves off of your/i, /you manage to loosen/i]]
[appraisal: for command remove my weapon.harness]
```

**Wear**:
```
[appraisal]>wear my weapon.harness
You toss one strap of the harness over your shoulder, pull it down and buckle it to its mate.
```
```
[appraisal: message: You toss one strap of the harness over your shoulder, pull it down and buckle it to its mate.]
[appraisal: checked against [/You sling/i, /You attach/i, /You .* unload/i, /You strap/i, /You slide/i, /You work your way into/i, /You spin/i, /You put/i, /You carefully loop/i, /slide effortlessly onto your/i, /You slip/i, /^You .* (?i-mx:\bweapon.*\bharness)/i]]
[appraisal: for command wear my weapon.harness]
```

**Comment by Nyctus** [comment]
**combat-trainer**

```
[combat-trainer: message: You deftly remove the diamondwood staff from your harness.]
[combat-trainer: checked against [/You get/i, /You pick/i, /You pluck/i, /You are already holding/i, /You fade in for a moment as you get/i, /I could not/i, /What were you/i, /Get what/i, /You need a free hand/i]]
[combat-trainer: for command get diamondwood.staff from my weapon harness]
[combat-trainer]>get my diamondwood.staff
```

**Comment by BinuDR** [comment]
Copy the file I edited over yours, run `;common-items`, and try again and see if this fixes this issue.  Here's a direct link to the edited version.

https://raw.githubusercontent.com/rpherbig/dr-scripts/423e9c22b83413a569ca6814cd984931ccac8a05/common-items.lic

For the other issue I'm going to need a harness to test with.

**Comment by Nyctus** [comment]
I might have extras with the same verbs, I can try to meet you in-game tomorrow after work (19:00 PDT)

Testing your change now!

**Comment by Nyctus** [comment]
I tested the changes to common-items.lic by overwriting and re-saving my local file but I ran into matchwait issues in appraisal.lic both when removing/equipping the harness and when retrieving/stowing the staff it was holding. (same as original issue)

Checked and I sadly I do not have a duplicate of this verby harness. I can lend it to you tomorrow and in the meantime I've swapped to a harness that doesn't have unique verbs.

Apologies for not realizing sooner the issue was this unique harness. Was my first time getting event gear and I didn't realize how much unique text all this stuff generates!
